clifford/_ast_transformer.py

```diff
@@ -0,0 +1,63 @@
+
+import ast
+
+
+class DecoratorRemover(ast.NodeTransformer):
+    """ Strip decorators from FunctionDefs"""
```

Comment:
```suggestion
    """ Strip decorators from top-level FunctionDefs"""
```

This doesn't recurse, but that's a good thing